### PR TITLE
[zh] Change i18n text in messages.xlf

### PR DESCRIPTION
**Pull request by windsonsea**
Changed for:

- consistency
- typos correction
- checked lines with `state="new"`

### CODE REVIEW

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf [hunk 1]
```diff
@@ -340,15 +340,15 @@
       </trans-unit>
       <trans-unit id="2027381191177847167" datatype="html">
         <source>Exec Commands</source>
-        <target state="new">Exec Commands</target>
+        <target state="new">执行命令</target>
```
Comment (hwdef) [inline modules/web/i18n/zh-Hans/messages.zh-Hans.xlf:343]:
```suggestion
        <target>执行命令</target>
```

Please remove `state="new"` for translated places.

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf [hunk 2]
```diff
@@ -464,7 +464,7 @@
       </trans-unit>
       <trans-unit id="2304348476142864790" datatype="html">
         <source>Persistent Volume Claims</source>
-        <target state="new">Persistent Volume Claims</target>
+        <target state="new">持久卷申领</target>
```
Comment (hwdef) [inline modules/web/i18n/zh-Hans/messages.zh-Hans.xlf:467]: Dedicated words are best left as they are

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf [hunk 3]
```diff
@@ -564,11 +564,11 @@
       </trans-unit>
       <trans-unit id="2733722431826186016" datatype="html">
         <source>Storage Classes</source>
-        <target state="new">Storage Classes</target>
+        <target state="new">存储类</target>
```
Comment (hwdef) [inline modules/web/i18n/zh-Hans/messages.zh-Hans.xlf:567]: Dedicated words are best left as they are

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf [hunk 4]
```diff
@@ -952,7 +952,7 @@
       </trans-unit>
       <trans-unit id="3787293861158300741" datatype="html">
         <source>Init containers</source>
-        <target>Init containers</target>
+        <target>Init 容器</target>
```
Comment (hwdef) [inline modules/web/i18n/zh-Hans/messages.zh-Hans.xlf:955]: Dedicated words are best left as they are

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf [hunk 5]
```diff
@@ -1104,7 +1104,7 @@
       </trans-unit>
       <trans-unit id="434404492084234684" datatype="html">
         <source>Volume</source>
-        <target>Volume</target>
+        <target>卷</target>
```
Comment (hwdef) [inline modules/web/i18n/zh-Hans/messages.zh-Hans.xlf:1107]: Dedicated words are best left as they are

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf [hunk 6]
```diff
@@ -1626,7 +1626,7 @@
       </trans-unit>
       <trans-unit id="6462224380417375328" datatype="html">
         <source>Service Accounts</source>
-        <target state="new">服务账号</target>
+        <target state="new">服务账户</target>
```
Comment (hwdef) [inline modules/web/i18n/zh-Hans/messages.zh-Hans.xlf:1629]: Dedicated words are best left as they are

**Review by hwdef — COMMENTED** [review]
Sorry for not receiving the notification, if you submit pr again in the future, please at me

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf (hunk 1 above)
Comment (windsonsea) [reply modules/web/i18n/zh-Hans/messages.zh-Hans.xlf]: Do I need to remove all `state="new"` in this file?

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf (hunk 6 above)
Comment (windsonsea) [reply modules/web/i18n/zh-Hans/messages.zh-Hans.xlf]: We have both `服务账号` and `服务账户` for `service account`. Changed all to `服务账号` now.

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf (hunk 1 above)
Comment (hwdef) [reply modules/web/i18n/zh-Hans/messages.zh-Hans.xlf]: If the line has already been translated and does not require modification,please remove the `state="new"`,If you have enough time, you can do some cleanup for this file.

Comment (windsonsea) [reply modules/web/i18n/zh-Hans/messages.zh-Hans.xlf]: Well received. Have checked all lines with `status="new"`

File: modules/web/i18n/zh-Hans/messages.zh-Hans.xlf (hunk 6 above)
Comment (hwdef) [reply modules/web/i18n/zh-Hans/messages.zh-Hans.xlf]: I think we should keep the `service account` in all place.

**Review by hwdef — COMMENTED** [review]
Thank you for your contribution to this project! 
In addition, I have two suggestions: 
1. Compile and deploy after the modification, and check that the modification is correct. 
2. Proper nouns remain as they are

**Pull request merged**